testgrid/specs/os-full.yaml

```diff
@@ -54,6 +54,22 @@
     curl -L -o centos2ol.sh https://raw.githubusercontent.com/oracle/centos2ol/main/centos2ol.sh
     chmod +x centos2ol.sh
     bash centos2ol.sh -r
+- id: rhel-9
+  name: RHEL
+  version: "9.x"
+  vmimageuri: https://download.rockylinux.org/pub/rocky/9/images/x86_64/Rocky-9-GenericCloud-Base.latest.x86_64.qcow2
+- id: centos-9
+  name: CentOS
+  version: "9.x"
+  vmimageuri: https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Stream-GenericCloud-9-20230227.0.x86_64.qcow2
```

Comment:
Do we actually want to test centos 9 stream?

Reply:
removed